data/encounters/encounter.json

```diff
@@ -0,0 +1,9 @@
+{
+  "encounter":  {
+    "name": "Goblin Attack!",
+    "monsters": [
+      {"race": "goblin", "max_health": "8", "name": "Goblin 1"},
```

Comment:
Let's put the monster name first so that the JSON file is easier to read through.